scaling_performance/optimizing_storage.adoc

```diff
@@ -175,13 +175,53 @@ to nodes to support a healthy cluster.
 == Choosing a Docker Graph Driver
 
 Docker stores images and containers in a graph driver (a pluggable storage
```

Comment:
Anywhere we can start to replace "Docker" with "Container Runtimes" it woiuld be helpful going forward, especially where we have started to release CRI-O